numpy/core/src/multiarray/mapping.c

```diff
@@ -727,8 +727,11 @@ prepare_index(PyArrayObject *self, PyObject *index,
         }
     }
     else if (used_ndim > PyArray_NDIM(self)) {
-        PyErr_SetString(PyExc_IndexError,
-                        "too many indices for array");
+        PyErr_Format(PyExc_IndexError,
+		     "array has %"NPY_INTP_FMT
```

Comment:
Still indentation error: which editor are you using ?

Reply:
vim...

Reply:
In local, there's no indent error. That's really weird...

![image](https://user-images.githubusercontent.com/4524261/77511910-560c3d00-6eb5-11ea-8911-c36521eb9291.png)